apps/_components/src/Showcase/Showcase.tsx

```diff
@@ -31,13 +31,13 @@ import { type HTMLAttributes, useState } from 'react';
 import classes from './Showcase.module.css';
 
 const DATA_PLACES = [
+  'Lillestrøm',
   'Sogndal',
   'Oslo',
   'Brønnøysund',
   'Stavanger',
   'Trondheim',
   'Bergen',
-  'Lillestrøm',
 ];
```

Comment:
This causes unwanted visual changes in our testing. You can move it up, but not into the first 4 slots.